src/utils/fetchSupabase.ts

```diff
@@ -0,0 +1,19 @@
+import { createClient } from '@supabase/supabase-js';
+
+export const SUPABASE_URL = 'https://rqbzgjjytqgzslodlusq.supabase.co';
+
+const supabaseKey = import.meta.env.VITE_SUPABASE_KEY as string;
+export const supabase = createClient(SUPABASE_URL, supabaseKey);
+
+//dobry typ ts objJson?>
```

Comment:
tak jak wcześniej pisałem przerobiłbym to żeby tu zawsze w środku było parsowanie. Wtedy tutaj będziesz dostawał object czyli Record<string,number|boolean|string>